slidedeck/src/slides/02-react-basics/00-react-basics.pug

```diff
@@ -171,18 +179,15 @@ section
   p These methods are called when an instance of a component is being created and inserted into the DOM:
   ol
     li <span class="component red">constructor()</span>
-    li <span class="component red">static getDerivedStateFromProps()</span>
```

Comment:
Note: I just removed these. I don't think they're really worth explaining, especially in an introductory course.